src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/NameQuotedValuePairListEncoding.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.Designer;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug
+{
+
+    [Export(typeof(INameValuePairListEncoding))]
+    [ExportMetadata("Encoding", "NameQuotedValuePairListEncoding")]
+    internal sealed class NameQuotedValuePairListEncoding : INameValuePairListEncoding
+    {
+        public IEnumerable<(string Name, string Value)> Parse(string value)
+        {
+            if (string.IsNullOrWhiteSpace(value))
+            {
+                yield break;
+            }
+
+            foreach (var entry in ReadEntries(value))
+            {
+                var (entryKey, entryValue) = SplitEntry(entry);
+
+                if (!string.IsNullOrEmpty(entryKey))
```

Comment:
return values of `SplitEntry` doesn't return null or empty. This check is not necessary.